src/asynchronous/server.rs

```diff
@@ -171,6 +171,15 @@ impl Server {
         Ok(())
     }
 
+    pub fn accept(&mut self, conn: Socket) -> Connection<impl Builder> {
```

Comment:
I think name it 'accept' is not that suitable because the input parameter conn is produced by the traditional accept(2).
It looks like this function will run after the accept(2). How about calling it like 'new_connection'

Reply:
Initially I had `serve_connected`, but I liked accept because it was the second thing I though about. While it is not a regular socket accept, it represents the functionality of an accept, but on a higher level. `new_connection` would also be fine for me.